src/Thenewboston/Bank/Models/ConnectionMessage.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Thenewboston.Bank.Models
+{
+    public class ConnectionMessage

```

Comment:
As @joshman1019 pointed out, it would be more clear to name this ConnectionRequestMessage. Otherwise, great work!

Reply:
Okay. Will do the necessary changed. Thanks @joshman1019  @Mirch